src/back/auth/src/service/service.cpp

```diff
@@ -57,8 +57,8 @@ Service::Service(
 	, _itemsLimitForList{conf["items-limit-for-list"].As<int>()}
 	, _tokenizer{ctx.FindComponent<Tokenizer>()}
 	, _oidc{ctx.FindComponent<OIDConnect>()}
-	, _rep{ctx.FindComponent<Repository>()}
-	, _session{_rep.Session(), _tokenizer.Session()}
+	, _rep{ctx.FindComponent<RepositoryComponent>()}
+	, _session{_rep.Session(), _tokenizer.Session(), _rep}
```

Comment:
Если мы передаём _rep у которого есть Session, то нам не нужно отдельно передавать _rep.Session()
```suggestion
	, _session{_rep, _tokenizer.Session()}
```

Reply:
Поправил (как понял).